Python/Product/Pylance/Templates/NewItem/mspythonconfig.json

```diff
@@ -0,0 +1,4 @@
+{
+    "$schema": "https://pvsc.blob.core.windows.net/mspythonconfig-schema/pyrightconfig-schema.json",
+    ""
```

Comment:
This is invalid JSON. If the user adds the item and doesn't edit it, it will cause an error on the next Pylance startup/update.
```
Error: Config file "C:\Projects\Temp\OpenFolderApp\mspythonconfig.json" could not be parsed. Verify that JSON is correct.
```

How about we put a real setting there as an example? Like:
"typeCheckingMode": "basic"

Reply:
Actually, `""` should be removed, leaving just the schema.